test/status-handler.test.js

```diff
@@ -229,15 +282,20 @@ describe('Status-Handler', function () {
     });
 
     it('should handle initial event, queued and complete event for 1 build', async function () {
+      const webhooksNock = nock('https://example.com')
+        .post('/webhooks')
+        .reply(204);
+
       const { Status, StatusHead, StatusEvent, StatusCounter } = handler.models;
       const spec = handler._transform(fixtures.singleQueued, 'counter');
-      await handler.event(fixtures.singleEvent);
       await handler.queued(fixtures.singleQueued);
+      await handler.event(fixtures.singleEvent);
```

Comment:
this event was intentionally before queued since npm install happens before events are queued. We should be adding ANOTHER event AFTER `handler.queued` is called which mimics the actual first event carpenter-worker sends so we can check against that.